src/snowcli/cli/nativeapp/run.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional
+from click.exceptions import ClickException
+from strictyaml import YAML
+
+
+class ApplicationPackageAlreadyExistsError(ClickException):
+    def __init__(self, name: str):
+        super().__init__(
+            f"An Application Package {name} already exists in account that may have been created without snowCLI. "
+        )
+
+
+class SnowflakeSQLExecutionError(ClickException):
```

Comment:
Does this belong in `nativeapp/` or is it more generic?

Reply:
I have moved this out to a top level exceptions class, I think it makes sense there.